mantle/cmd/ore/gcloud/update-image.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcloud
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/coreos/mantle/platform/api/gcloud"
+)
+
+var (
+	cmdUpdate = &cobra.Command{
+		Use:   "update-image --image=ImageName [--family=ImageFamily] [--description=ImageDescription]",
```

Comment:
Generally the `Use` field is only the command name and any parameters that do not have a flag.

Reply:
Can you be more specific? What do you suggest I change it to?

Reply:
Just `update-image`.

Reply:
simple enough. I was thinking the text was adding value by indicating which "options" were optional and which ones weren't optional

Reply:
will be fixed in next upload

Reply:
fixed now